jakartaee-8-archetype/src/main/resources/archetype-resources/buildAndRun.sh

```diff
@@ -1,3 +1,6 @@
 #!/bin/sh
-mvn clean package && docker build -t ${groupId}/${artifactId} .
-docker rm -f ${artifactId} || true && docker run -d -p 9080:9080 -p 9443:9443 --name ${artifactId} ${groupId}/${artifactId}
\ No newline at end of file
+if [ "$(docker ps -a -f name=${artifactId} | grep -w ${artifactId} | wc -l)" -eq 1 ]
+then
+  mvn clean package && docker build -t ${groupId}/${artifactId} .
+fi
+docker rm -f ${artifactId} || true && docker run -d -p 9080:9080 -p 9443:9443 --name ${artifactId} ${groupId}/${artifactId}
```

Comment:
shouldn't only `docker rm -f ${artifactId} ` be inside the if? To only run `docker rm` if we find an existing container with the same name?

Reply:
Oh yeah you are right, strange that this worked yesterday 🙈